main.cpp

```diff
@@ -284,6 +300,11 @@ int Keyboard::buttonPressTime()
     return time;
 }
 
+void Keyboard::buttonColorCout()
```

Comment:
Name things what they are. 
Name functions based on what they do, and make sure that the name includes a verb
- [ ] revise